server/src/test/java/keywhiz/service/permissions/AlwaysAllowDelegatingPermissionCheckTest.java

```diff
@@ -24,8 +23,8 @@ public class AlwaysAllowDelegatingPermissionCheckTest {
   private MetricRegistry metricRegistry;
   private AlwaysAllowDelegatingPermissionCheck alwaysAllow;
 
-  private static Objects target;
-  private static Objects principal;
+  private static Object target = new Object();
+  private static Object principal = new Object();
 
```

Comment:
I ran into an issue where (before renaming our new isAllowed overload) Mockito got confused because null is a sort of supertype-of-all-types and it wouldn't match properly. Using non-null values fixed this and I think will help ensure more correct behavior going forward.